site/_posts/2021-06-28-identiverse-conference-report.md

```diff
@@ -0,0 +1,90 @@
+
+- Manages an open technical specification for strong passwordless authentication.
+- Runs a certification program allowing companies to certify their solution.
+- Runs programs to facilitate adoption of the standard. According to their executive director and CMO, this is an increasing portion of their effort now.
+
+I attended one session where presenters from Intuit and T-Mobile talked about rollouts of FIDO enabled applications. This panel was a continuation of a discussion with the same attendees two years ago, which is an interesting way to close the loop at a conference.
+
+FIDO adoption doesn't necessarily mean the death of passwords. But during this rollout they found large numbers of users (>80%) preferred to authenticate with FIDO enabled methods. T-Mobile uses FIDO to authenticate you before a call to customer service via 611 connects. Intuit has rolled it out across their major mobile applications.
+
+FIDO also plays in continuous evaluation of authorization. The T-Mobile representative stated the discussion was evolving from a "what is the best second factor discussion," which was common a few years ago, to a multi-factor world where you are mapping the right set of authenticators for the current user action. Some situations call for strong recent authentication, such as changing payment information, while others, like checking the minutes, require less assurances.
+
+### The rest of it
+
+There was a lot of other content which didn't fall into one of the two common themes above:
+
+* An [IAM certification program](https://idpro.org/cidpro/) was announced.
+* There was more than one diversity and inclusion discussion.
+* An interesting talk on OAuth and secure mobile applications
+* An overview of all the ways you can authenticate with AWS
+* And more.
+
+One surprise to me was how little code I saw. Even in the mobile application security session, I didn't see much code displayed. Maybe I simply missed the code-heavy sessions, but this seemed like a conference for IAM admins rather than devs.
+
+However, it was fantastic for learning more about the wide world of IAM. I talked to many people who were doing interesting things with identity. One wanted to leverage DNS to enable a rollout of digital identity. I got a chance to chat in person with a couple of the IAM personalities that are "big names" in the space.
+
+I also sat next to a person at breakfast who ran identity for over one hundred hospitals and had some unique challenges. We talked a bit about how he planned rollouts in the hospital environment; these are sensitive environments in so many respects.
+
+Another attendee I chatted with handled an enterprise identity integration project during a merger of several hospitals through a solution which allows creation of virtual LDAP directories. After a physical LDAP directory merger was required due to EMR compliance concerns, they still use virtual LDAP directories to control access for SaaS applications.
```

Comment:
Another attendee I chatted with handled an enterprise identity integration project during a merger of several hospitals through a solution which allows creation of virtual LDAP directories

This sense is a bit unwieldy